packages/react-strict-dom/tests/__snapshots__/css-test.native.js.snap-native

```diff
@@ -454,75 +454,171 @@ exports[`properties: general textShadow 2`] = `
 exports[`properties: general transform: matrix 1`] = `
 {
   "style": {
-    "transform": "matrix(0.1, 1, -0.3, 1, 0, 0)",
+    "transform": [
+      {
+        "matrix": [
+          0.1,
+          1,
+          0,
+          0,
+          -0.3,
+          1,
+          0,
+          0,
+          0,
+          0,
+          1,
+          0,
+          0,
+          0,
+          0,
+          1,
+        ],
+      },
+    ],
   },
 }
 `;
 
 exports[`properties: general transform: mixed 1`] = `
 {
   "style": {
-    "transform": "
-          rotateX(1deg) rotateY(2deg) rotateZ(3deg) rotate3d(1deg, 2deg, 3deg)
-          scale(1) scaleX(2) scaleY(3) scaleZ(4) scale3d(1,2,3)
-          translateX(1px) translateY(1em) translateZ(1rem) translate3d(1px, 1em, 1rem)
-        ",
+    "transform": [
+      {
+        "rotateX": "1deg",
+      },
+      {
+        "rotateY": "2deg",
+      },
+      {
+        "rotateZ": "3deg",
+      },
+      {
+        "scale": 1,
+      },
+      {
+        "scaleX": 2,
+      },
+      {
+        "scaleY": 3,
+      },
+      {
+        "scaleZ": 4,
+      },
+      {
+        "translateX": 1,
+      },
+    ],
```

Comment:
The loss of `translateY` here isn't technically a regression because the value is `1em` and wouldn't have worked before. (And the other transform types aren't supported on React Native). However we could look into supporting non-px units for translate at some point in the future